src/Microsoft.ML.Api/Microsoft.ML.Api.csproj

```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup>
      <PackageReference Include="System.CodeDom" Version="$(SystemCodeDomPackageVersion)" />
+     <PackageReference Include="System.Memory" Version="4.5.1" />
```

Comment:
These references flow transitively through project-to-project references. So you should only need it on `Microsoft.ML.Core.csproj`, and all the other .csprojs will automatically get the `PackageReference`.